packages/ERTP/src/issuerKit.js

```diff
@@ -322,7 +322,7 @@ harden(prepareIssuerKit);
  *   anything else is corrupted by that corrupted state. See
  *   https://github.com/Agoric/agoric-sdk/issues/3434
  * @param {IssuerOptionsRecord} [options]
- * @returns {IssuerKit<K, any>}
+ * @returns {IssuerKit<K>}
```

Comment:
Why not also parameterize the type of `makeIssuerKit` with the thing you're currently calling `M`? Is this where the inference on the `elementShape` would go?

Reply:
just a matter of prioritization. done now